### PR TITLE
Add rubygem-pulp_python_client package

### PR DESCRIPTION
**Pull request by ianballou**
Adds pulp_python_client in preparation for adding the pulp_python repository type to Katello.

### CODE REVIEW

File: comps/comps-foreman-el8.xml [hunk 1]
```diff
@@ -257,6 +257,7 @@
       <packagereq type="default">rubygem-prometheus-client</packagereq>
       <packagereq type="default">rubygem-promise.rb</packagereq>
       <packagereq type="default">rubygem-public_suffix</packagereq>
+      <packagereq type="default">rubygem-pulp_python_client</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-el8.xml:260]: I'd think these need to go to katello comps? :)

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -248,6 +248,7 @@
       <packagereq type="default">tfm-rubygem-prometheus-client</packagereq>
       <packagereq type="default">tfm-rubygem-promise.rb</packagereq>
       <packagereq type="default">tfm-rubygem-public_suffix</packagereq>
+      <packagereq type="default">tfm-rubygem-pulp_python_client</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-rhel7.xml:251]: I'd think these need to go to katello comps? :)

**Review by evgeni — CHANGES_REQUESTED** [review]

File: comps/comps-foreman-el8.xml (hunk 1 above)
Comment (ianballou) [reply comps/comps-foreman-el8.xml:260]: Oh, I missed that. I used `./add_gem_package.sh pulp_python_client:3.2.0 scl katello-nightly-rhel7`. Maybe there's a bug with the script.

Comment (evgeni) [reply comps/comps-foreman-el8.xml]: It should. @ekohl did some changes recently, maybe that backfired?

**Comment by ianballou** [comment]
@evgeni I think we're all good now.

**Review by evgeni — APPROVED** [review]
ACK

I don't have my koji creds here, so you might need to ping @jlsherrill or @ehelms  for merging ;)

**Pull request merged**